storage/postgres/postgres_adapter_test.go

```diff
@@ -49,7 +50,7 @@ func setup() *dockertest.Resource {
 	}
 
 	// Pulls an image, creates a container based on it and runs it
-	resource, err := pool.Run("postgres", "latest", []string{"POSTGRES_HOST_AUTH_METHOD=trust", fmt.Sprintf("POSTGRES_DB=%s", testDbName)})
+	resource, err := pool.Run("postgres", "13", []string{fmt.Sprintf("POSTGRES_PASSWORD=%s", testPassword), fmt.Sprintf("POSTGRES_DB=%s", testDbName)})
```

Comment:
This should make testing more realistic and more stable. It might be out of scope for this post, but maybe a part 2 could show how to test multiple DB versions? Since `t.Run`s can be nested, you could have a table of different container initializer functions, and within that a table of test cases. The test output is nice, and it's really great for expressing (and testing) support for different DB versions.